src/deltacode/cli.py

```diff
@@ -51,12 +51,15 @@ def write_json(deltacode, outfile, all_delta_types=False):
         ('delta_stats', deltacode.stats.to_dict()),
         ('deltas', deltas(deltacode, all_delta_types))
     ])
-
+    click.secho(get_notice(),fg="green")  
+    click.echo('deltas_count: {}'.format(len([d for d in deltas(deltacode, all_delta_types)])))
+    click.echo('delta_stats: ')
+    for stat in deltacode.stats.to_dict():
+        click.echo("""   "{}": {}""".format(str(stat),deltacode.stats.to_dict()[stat]))  
```

Comment:
I do not think we want these unnecessary spaces in the output. Also, I would like the "stats" to be broken down by "factors" as this is more detailed.

Reply:
@majurg ,I was actually providing the old_files_count ,files_modifies,files_moves....
in the stats part
What else should we provide ?
should we also provide the stats of new and old file path as it appears in the headers of the delta code json file?

Reply:
I would like the "stats" to be broken down by "factors" as this is more detailed.

Reply:
Yes we can provide the factors of each and every deltas in the cli ,and that is actually much elaborate ,however some deltas may be having empty arrays as its "factors" (in case of "added"...) ,what do we need to provide as "factors" in such a case?

Reply:
Then you just use the base `status` value.

Reply:
Yes @majurg I think we can provide both the status and factors for the deltas as that will actually much detailed

Reply:
@majurg can you please review the changes.